graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -671,7 +671,7 @@ def test_graph_relationshipProperties_stream_with_arrow(gds: GraphDataScience) -
     assert {e for e in y_values["propertyValue"]} == {5, 6, 7}
 
 
-def test_graph_streamRelationshipProperties_with_arrow_separate_property_columns(gds: GraphDataScience) -> None:
+def test_graph_streamRelationshipProperties_separate_property_columns_dep(gds: GraphDataScience) -> None:
```

Comment:
I think we want to keep `with_arrow` in the name. It uses `gds`, that has arrow enabled. `gds_without_arrow` is the one without.